wercker.yml

```diff
@@ -1,29 +1,33 @@
+---
```

Comment:
I think we can remove the wercker file, since the GitHub actions will now do the build/test/lint?

Reply:
No problem, I wasn't sure if you wanted to keep it as a redundancy measure. 👍